qlib/rl/order_execution/strategy.py

```diff
@@ -89,6 +89,7 @@ def __init__(
         exchange: Exchange,
         ticks_per_step: int,
         backtest_data: IntradayBacktestData,
+        data_granularity: int = 1,
```

Comment:
Maybe data_granularity should be a property of IntraDayBacktestData?

Reply:
Personally think it is better to leave it as it is after several attempts. We can fix this in the future if we find a better design.